pkgs/applications/audio/pavucontrol/default.nix

```diff
@@ -19,7 +19,7 @@ stdenv.mkDerivation rec {
   buildInputs = [ libpulseaudio gtkmm3 libcanberra-gtk3 makeWrapper
```

Comment:
You should remove `makeWrapper` and `preFixup` since that is already handled by `wrapGAppsHook`.

Reply:
Thanks, done